beets/autotag/mb.py

```diff
@@ -156,6 +156,11 @@ def track_info(recording, index=None, medium=None, medium_index=None,
     ``medium_index``, the track's index on its medium; ``medium_total``,
     the number of tracks on the medium. Each number is a 1-based index.
     """
+    flag = False
+
+    if recording['title'] == '[silence]':
+        flag = True
```

Comment:
Instead of this pattern:

```py
some_variable = False
if some_boolean:
    some_variable = True
```

It's simpler to just write:

```py
some_variable = some_boolean
```